src/datatypeconvertor.js

```diff
@@ -213,6 +216,28 @@ $.DataTypeConvertor = class {
             return context;
         };
 
+        this.learn("rasterBlob", "image", (tile, blob) => new $.Promise((resolve, reject) => {
```

Comment:
We can also talk about what are the best type names. `rasterBlob` or `blob` -> if we want to stress that the binary data is a raster image, or not. Or, we could get inspired by e.g. HTTP response and have pair data type + mime type...

Reply:
rasterBlob sounds good to me. Nice to be a little specific. Maybe we'll have other kinds of bobs in the future?

Would be good to have this type documented somewhere.

Reply:
Yes, definitely. They are here. https://github.com/openseadragon/site-build/pull/289/files#diff-70a0f23b479a21812f4f1d805a5e03458940c0fd7786e0a73733c9b2d45577ea Maybe we could pull them out to some more visible place...

Reply:
Would it make sense to have them somewhere in the doc comments?

I'm not seeing `rasterBlob` in that list. Should it be there?

Reply:
Yes it should, I just did not update it yet.

Reply:
Cool, don't forget to!